Admin-app/src/components/SignIn/index.js

```diff
@@ -183,7 +180,22 @@ export default function SignIn() {
                 Error
               </Typography>
               <Typography id="modal-modal-description" sx={{ mt: 2 }}>
-                You are currently in the Administration Application, please sign in using this email on the Client App.  
+                This email is not registered with the Administration application.
+              </Typography>
+            </Box>
+          </Modal>
+          <Modal
```

Comment:
@tusharraval102  can you create a component for this  please?

Reply:
Yup, ill do that now